lib/Elastica/Filter/GeoBoundingBox.php

```diff
@@ -39,15 +39,15 @@ public function __construct($key, array $coordinates)
      */
     public function addCoordinates($key, array $coordinates)
     {
-        if (!isset($coordinates[0]) || !isset($coordinates[1])) {
-            throw new InvalidException('expected $coordinates to be an array with two elements');
-        }
+        if (isset($coordinates[0], $coordinates[1])) {
```

Comment:
I liked the previous code more because the normal behaviour is on the top level of the function and special checks are made for the exception.